src/relay/limiter/fixed.rs

```diff
@@ -5,19 +5,22 @@ use lru::LruCache;
 use std::{sync::Arc, time::UNIX_EPOCH};
 use tokio::sync::Mutex;
 
+/// FixedWindowOptions are used to determine user limits for each time window, and the size of that window.
 pub struct FixedWindowOptions {
     pub size: usize,
```

Comment:
what does this size represent? is it the message size ? or something else? and in which unit?

Reply:
size represents the total size of the messages in bytes.
count represents the number of messages.
window represents the window size in seconds.

I added this to the docs also.